content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">Infrastructure Vulnerability Management is in beta.</div>
+
+## Overview
+
+Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, and manage vulnerabilities in your organization. Infrastructure Vulnerability Management offers visibility into the vulnerabilities in the [Container Images][2] and hosts in your infrastructure.
```

Comment:
@aliciascott let's change this to: Datadog Infrastructure Vulnerability Management combines observability data with security insights to detect, prioritize and manage vulnerabilities in your organization. It offers visibility into the vulnerabilities in the [container images] [2] and [hosts] [We should link to this as well] in your infrastructure.

Reply:
Updated